activescott/serverless-aws-static-file-handler
# Chore: move package files to root

## What did you implement:

Closes #21



## How did you implement it:
Just moved the files, and then updated some path references


## How can we verify 
* Run `npm run test` and see all is green
* Edit a file, like a small lintable change (add space, add a semi-colon...), and then run `npm run lint` and `npm run lint-fix`



## Todos:

~- [ ] Write tests~
- [x] All tests pass on our CI Server: https://travis-ci.org/activescott/serverless-aws-static-file-handler
